rmf_site_editor/Cargo.toml

```diff
@@ -51,6 +51,10 @@ ehttp = { version = "0.4", features = ["native-async"] }
 nalgebra = "0.32.5"
 anyhow = "*"
 
+[features]
+include-run-js = []
+default = ["include-run-js"]
```

Comment:
As discussed in the PMC session, I'd suggest either
1. don't make this a default feature (meaning we need to explicitly enable the feature [here](https://github.com/open-rmf/rmf_site/blob/ae04fcbb5f5b1abfa27682ef15884afa9137ffd2/scripts/build-web.sh#L6)) or
2. make a new `rmf_site_[editor_]web` crate that's just a thin wrapper around `rmf_site_editor` with the addition of the `run_js` function.

Reply:
Made the feature default off https://github.com/open-rmf/rmf_site/pull/249/commits/7afe92f0f163c71d6c7df361e028ec7da4bf395c

Reply:
After thinking about it some more, making it as a separate crate would have the added benefit of providing an explicit example to downstream users for how to make their own custom web app. Would that be worth the added effort?

Reply:
Done in https://github.com/open-rmf/rmf_site/pull/249/commits/320f362546a808c274d53a8c5cbe648a1fab5d4e, I was a bit on the fence on bikeshedding `rmf_site_web` vs `rmf_site_editor_web`, the second is more accurate but adds additional typing when trying to navigate the workspace with autocompletion, still went with `rmf_site_editor_web` in the end.
There is probably more splitting we could do to reduce the amount of `cfg` flags in the main app but this is the minimal "embed site editor" shim